packages/editor/src/components/post-taxonomies/index.js

```diff
@@ -25,11 +20,10 @@ export function PostTaxonomies( {
 	taxonomies,
 	taxonomyWrapper = identity,
 } ) {
-	const availableTaxonomies = filter( taxonomies, ( taxonomy ) =>
+	const availableTaxonomies = ( taxonomies ?? [] ).filter( ( taxonomy ) =>
```

Comment:
Although very edge case-y, but `taxonomies` could be `null`:

https://github.com/WordPress/gutenberg/blob/33d84b036592a5bf31af05b7710f3b2b14163dc4/packages/core-data/src/queried-data/selectors.js#L99